proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -75,6 +76,21 @@ message RepositoryContributor {
   RepositoryRole implicit_role = 4;
 }
 
+message RepositoryMetadata {
```

Comment:
We also have [RepositoryCounts](https://github.com/bufbuild/buf/blob/main/proto/buf/alpha/registry/v1alpha1/repository.proto#L59-L65). Are they going to stay seperate? Seems like they would belong here.